README.md

```diff
@@ -70,3 +70,16 @@ Nevertheless you are able to override them:
       ],
     ];
 ```
+
+### Themes
+The select2 library provides the possibility to use your own [theme](https://select2.org/appearance#themes) for the select2 widgets. This module provides a
```

Comment:
This link does not really explain much, we should state the necessary structure of the css file (e.g. naming of the css classes with the theme name as suffix)